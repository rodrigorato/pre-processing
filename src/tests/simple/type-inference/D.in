// The original code was retrieved from https://docs.oracle.com/javase/tutorial/java/javaOO/anonymousclasses.html
public class Foo {
    public static void main(String[] args) {
		var foo = new Foo() {
            public void bar() {
            	 var name = new String("Baz");
                 System.out.println("Quux " + name);
            }
        };
    }
}