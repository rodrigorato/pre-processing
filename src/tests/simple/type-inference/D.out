// The original code was retrieved from https://docs.oracle.com/javase/tutorial/java/javaOO/anonymousclasses.html
public class Foo {
    public static void main(String[] args) {
		Foo foo = new Foo() {
            public void bar() {
            	 String name = new String("Baz");
                 System.out.println("Quux " + name);
            }
        };
    }
}